hera_cal/tests/test_vis_clean.py

```diff
@@ -173,6 +262,12 @@ def test_fft_data(self):
         pytest.raises(ValueError, V.fft_data, keys=[])
         pytest.raises(ValueError, V.fft_data, keys=[('foo')])
 
+    # THIS UNIT TEST IS BROKEN!!!
+    # It was cleaning with one dnu and trimming with another.
+    # I am preserving this bad behavior but when I remove the bad
+    # behavior it fails. Someone should fix this unit test.
+    # I don't think it should be done in this PR though -- whose goal is to
```

Comment:
These kind of comments belong in an issue, not in the code.

Reply:
I also created an issue at the same time. https://github.com/HERA-Team/hera_cal/issues/603

I'll reference the issue here as well.